tests/substrate_tests/expressions.rs

```diff
@@ -247,21 +247,19 @@ fn expressions() {
 }
 
 #[test]
-#[should_panic]
 fn divisions_by_zero() {
     // parse
     let mut runtime = build_solidity(
         "
         contract test {
-            function do_test() public returns (uint){
-                uint256 val = 100;
-
-                return (val / 0);
+            uint256 divisor = 0;
+            function do_test(uint256 divisor) public returns (uint256 result){
+                result = 100 / divisor;
```

Comment:
The argument is called `divisor` and you have a storage variable named `divisor`. This test seems a bit awkward because of the naming conflict.

Reply:
Very good catch, thanks!